test/parallel/test-whatwg-url-tojson.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const URL = require('url').URL;
+
+// Tests below are not from WPT.
+const url = new URL('http://example.org');
```

Comment:
I'm asking because it seems complicated to keep the tests up to date with WPT tests if we have to copy/paste them by hand.